docs/news/governance.md

```diff
@@ -1,27 +1,35 @@
+:-----------:|:-----------:|:-----------:|
+[_Governance_](governance.md) | [_Governance_](governance.md) | [_Governance_](governance.md) |
+[<img src="https://images.unsplash.com/photo-1515040242872-08257d6d08c2?ixlib=rb-0.3.5&s=e7e07b35570d150a85700c871139f1fb&auto=format&fit=crop&w=800&q=80">](https://marginalrevolution.com/marginalrevolution/2018/06/towards-international-court-smart-contract-arbitration.html) | [<img src="https://cdn-images-1.medium.com/max/800/1*kbBSEYx4ssictZbc2P3IgQ.jpeg">](https://blog.foam.space/exploring-ethereum-storage-costs-with-the-foam-developer-tools-96d84e1a06b5) | [<img src="https://images.unsplash.com/photo-1499909694555-1ae5b7067b1a?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=85c6f4950e33e2feb5252e160c649a21&auto=format&fit=crop&w=800&q=80">](https://medium.com/@richardred/what-is-on-chain-cryptocurrency-governance-is-it-plutocratic-bfb407ef6f1) |
+_Author [Alex Tabarrok](https://marginalrevolution.com/about) on Jun 14_ | _Author [Ilya Ostrovskiy](https://blog.foam.space/@iostat_foam) on Jun 18_ | _Author [Richard Red](https://medium.com/@richardred) on Jun 20_ |
+**Lex Cryptographia, the legal system of smart contracts and code-is-law** In this article, Alex proposes a court system analogous to the courts of arbitration that govern international commerce be created to govern smart contracts in virtual space. | **A good craftsman never has to blame their tools.** A showcase on how we used our stack to help drive the design and engineering process behind the FOAM TCR. | **A rebuttal on On-Chain Blockchain Governance** A detailed response to a number of posts from prominent blockchain personalities about how on-chain governance based on coin-holder votes is inherently plutocratic and bad. |
+[Read More](https://marginalrevolution.com/marginalrevolution/2018/06/towards-international-court-smart-contract-arbitration.html) | [Read More](https://blog.foam.space/exploring-ethereum-storage-costs-with-the-foam-developer-tools-96d84e1a06b5) | [Read More](https://medium.com/@richardred/what-is-on-chain-cryptocurrency-governance-is-it-plutocratic-bfb407ef6f1) |
+
+[**Voting and Governance in Security Tokens**](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [**Crypto Commons — from tragedy to wealth of the commons**](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
+:-----------:|:-----------:|:-----------:|
+[_Governance_](governance.md) | [_Governance_](governance.md) | |
+[<img src="https://cdn-images-1.medium.com/max/800/1*F7oHt_loAWz4EACe_SqRgQ.jpeg">](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [<img src="https://cdn.britannica.com/668x448/09/134309-004-ABF9FE28.jpg">](https://blog.usejournal.com/crypto-commons-da602fb98138) | |

```

Comment:
Last, empty, column could use the `../../images/monthly_no_image.png` image and the `Submit News!` link.